### PR TITLE
Deprecate HMI RPC OnFindApplications

### PR DESCRIPTION
**Pull request by JackLivio**
This proposal is for deprecating and removing an unusable HMI RPC `OnFindApplications`.

### CODE REVIEW

File: proposals/NNNN-Deprecate-HMI-RPC-OnFindApplications.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+```
```
Comment (theresalech) [inline proposals/NNNN-Deprecate-HMI-RPC-OnFindApplications.md]: These should be removed for proper formatting

File: proposals/NNNN-Deprecate-HMI-RPC-OnFindApplications.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
++     <description>DEPRECATED - This RPC is not implemented</description>
+      <param name="deviceInfo" type="Common.DeviceInfo" mandatory="false">
+        <description>The name and ID of the device the list of registered applications is required for.</description>
+      </param>
+    </function>
+```
+
+## Potential downsides
+
+An OEM could have implemented this RPC on their own in which case the Author requests that OEM's reviewing this proposal note if they are using this RPC.
```
Comment (theresalech) [inline proposals/NNNN-Deprecate-HMI-RPC-OnFindApplications.md]:
```suggestion
An OEM could have implemented this RPC on their own in which case the Author requests that OEMs reviewing this proposal note if they are using this RPC.
```

File: proposals/NNNN-Deprecate-HMI-RPC-OnFindApplications.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+## Impact on existing code
+
+Only impacts SDL Core. OnFindApplications should be removed from the HMI API and the OnFindApplications supporting command files should be removed from the repository.
```
Comment (theresalech) [inline proposals/NNNN-Deprecate-HMI-RPC-OnFindApplications.md]: Will this require a change to the SDL and/or Generic HMI?  If so, please describe here.  If not, please remove from `Impacted Platforms` section.

**Comment by JackLivio** [comment]
@theresalech Updated. Ready for re-review

**Pull request merged**